### PR TITLE
NO_JIRA: fix d-c setup

### PR DESCRIPTION
**Pull request by krdabrowski**

### CODE REVIEW

**Review by azakrzewski-hy — APPROVED** [review]
LGTM

File: distribution/src/main/resources/docker-compose/docker-compose-minimal.yml [hunk 1]
```diff
@@ -29,6 +29,8 @@ services:
         -Dhxi.client.base-url=http://hxinsight-mock:8080
         -Dhxi.auth.providers.hyland-experience.token-uri=http://hxinsight-mock:8080/token
         "
+    ports:
+      - "8080:8080"
```
Comment (mpichura) [inline distribution/src/main/resources/docker-compose/docker-compose-minimal.yml:33]: Is `8000:8000` from extended service still exposed/mapped?

Comment (krdabrowski) [reply distribution/src/main/resources/docker-compose/docker-compose-minimal.yml:33]: Yes, it is

**Review by mpichura — APPROVED** [review]
LGTM

**Pull request merged**